### ISSUE TITLE
Xamarin.Android.Crashlytics Please Sign Up

### ISSUE THREAD

**Issue by Dooks123** [xamarin/XamarinComponents#448]
I can not seem to get it to work. I have previously used `Xamarin.Firebase.Crash` but now that you have to link it to Fabric, I added the following NuGet packages
- Xamarin.Android.Crashlytics
- Xamarin.Android.Crashlytics.Core
- Xamarin.Android.Crashlytics.Beta
- Xamarin.Android.Crashlytics.Answers

and I have added the following Meta Tag according to the installation [https://fabric.io/kits/android/crashlytics/install](https://fabric.io/kits/android/crashlytics/install) in `AndroidManifest.xml` in the `<application..>` tag:

`<meta-data android:name="io.fabric.ApiKey" android:value="xxxMYKEYxxx" />`

And I have added this in `MainApplication.cs` `OnCreate()` method:

`Fabric.Fabric.With(this, new Crashlytics.Crashlytics());`

I get the following error:
```
Fabric.Services.Concurrency.UnmetDependencyException:
This app relies on Crashlytics. Please sign up for access at https://fabric.io/sign_up,
install an Android build tool and ask a team member to invite you to this app's organization.
```

Or this one:
```
Fabric.Services.Concurrency.UnmetDependencyException: <Timeout exceeded getting exception details>
09-10 12:10:52.914 W/Fabric  (12456): Could not call getAdvertisingIdInfo on com.google.android.gms.ads.identifier.AdvertisingIdClient
09-10 12:10:52.915 W/Fabric  (12456): Could not call getId on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
09-10 12:11:02.934 W/Fabric  (12456): Could not call getAdvertisingIdInfo on com.google.android.gms.ads.identifier.AdvertisingIdClient
09-10 12:11:02.936 W/Fabric  (12456): Could not call isLimitAdTrackingEnabled on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
09-10 12:11:03.180 W/zygote  (12456): Long monitor contention with owner Queue (12499) at java.lang.Class java.lang.Class.classForName(java.lang.String, boolean, java.lang.ClassLoader)(Class.java:-2) waiters=0 in io.fabric.sdk.android.services.common.AdvertisingInfo io.fabric.sdk.android.services.common.IdManager.getAdvertisingInfo()
```

I have obviously Signed-Up on `Fabric.io` because that is how I got the key in the first place.

Yes, this is a working app, I have all the necessary permissions in

**Comment by Dooks123** [comment]
No Worries. I got it to work.

I had to add the following line in the `Resources/values/Strings.xml` file as well:
`<string name="com.crashlytics.android.build_id">236</string>`

So for everybody else:
1. Add nuget packages
2. Add the MetaData line in AndroidManifest with your `API_KEY`
3. Add the string Resource, from above, `name="com.crashlytics.android.build_id"`
4. Add the `Fabric.Fabric.With(this, new Crashlytics.Crashlytics());` line in `MainApplication.cs`

It should run then.
